lib/matplotlib/testing/exceptions.py

```diff
@@ -1,15 +1,3 @@
-class KnownFailureTest(Exception):
```

Comment:
Yeah, I am pretty sure other projects actually uses our KnownFailure stuff.

On Wed, Aug 17, 2016 at 1:15 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In lib/matplotlib/testing/exceptions.py
> <https://github.com/matplotlib/matplotlib/pull/6920#discussion_r75165500>:
>
> > @@ -1,15 +1,3 @@
> > -class KnownFailureTest(Exception):
>
> I would rather not move this for compatibility reasons.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6920/files/f20efb99d67e581f18f8fd58db07d6279528fcbd#r75165500>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARy-A6W8iBehFuv_ltVBw91aSSmfrI6ks5qg0GmgaJpZM4Jemgm>
> .
>